tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,62 +159,99 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     };
 
     fluid.tests.tooltipModuleSource = function (pager) {
-        var pageLinksTop = $("a", pager.pagerBar.locate("pageLinks"));
-        var pageLinksBottom = $("a", pager["pagerBar-1"].locate("pageLinks"));
-
-        var tooltipContents = [
-            [
-                {nodeName: "b", nodeText: "bird"},
-                {nodeName: "b", nodeText: "fish"}
-            ]
-        ];
-
         var sequence = [];
+        var linksList = pager.dataModel.map(function(data) {
+            return {text: data.animal};
+        });
```

Comment:
Could you use the `fluid.transform` utility for this. http://docs.fluidproject.org/infusion/development/CoreAPI.html#fluid-transform-source-fn1-fn2-